src/redux/actions.js

```diff
@@ -19,8 +18,3 @@ export const listenToQueries = () => (
     });
   }
 );
-
-export const deleteVisible = (index) => ({
```

Comment:
Im probably going to write another action that will fire after the gsap animation completes